docs/addons.md

```diff
@@ -89,7 +89,15 @@ Read more about cluster autoscaler in the [official documentation](https://githu
 ##### Expander strategies
 Cluster autoscaler supports several different [expander strategies](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-are-expanders).
 
-Note that the `priority` expander requires additional configuration through a ConfigMap as described in [its documentation](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/expander/priority/readme.md) - you will need to create this ConfigMap in your cluster before selecting this expander.
+Note that the `priority` expander requires additional configuration through a ConfigMap as described in [its documentation](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/expander/priority/readme.md) - this ConfigMap will be generated based on instance group you can change priority of each instance group by adding the followig to instance group spec.
```

Comment:
This is missing the configmap creation flag, I think.
Also please add `{{ kops_feature_table(kops_added_default='1.26') }}`